batcher/aligned-sdk/src/errors.rs

```diff
@@ -0,0 +1,132 @@
+use core::fmt;
+use ethers::providers::ProviderError;
+use ethers::signers::WalletError;
+use ethers::utils::hex::FromHexError;
+use std::io;
+use std::path::PathBuf;
+
+pub enum AlignedError {
+    SubmitError(SubmitError),
+    VerificationError(VerificationError),
+}
+
+impl From<SubmitError> for AlignedError {
+    fn from(e: SubmitError) -> Self {
+        AlignedError::SubmitError(e)
+    }
+}
+
+impl From<VerificationError> for AlignedError {
+    fn from(e: VerificationError) -> Self {
+        AlignedError::VerificationError(e)
+    }
+}
+
+impl fmt::Debug for AlignedError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            AlignedError::SubmitError(e) => write!(f, "Submit error: {:?}", e),
+            AlignedError::VerificationError(e) => write!(f, "Verification error: {:?}", e),
+        }
+    }
+}
+
+pub enum SubmitError {
+    ConnectionError(tokio_tungstenite::tungstenite::Error),
+    IoError(PathBuf, io::Error),
+    SerdeError(serde_json::Error),
+    EthError(String),
+    SignerError(String),
+    PasswordError(io::Error),
```

Comment:
```suggestion
    PasswordError(io::Error),
```

Password error shouldn't be here right ? And maybe keystore ?

Reply:
Fixed in [593cb70](https://github.com/yetanotherco/aligned_layer/pull/459/commits/593cb70a1dba799eec834ebbbd2626624dd3f947).